app/src/main/java/org/mozilla/tv/firefox/ext/Js.kt

```diff
@@ -249,4 +249,28 @@ var _firefoxTV_isPlaybackStateObserverLoaded;
     }
 })();
 """.trimIndent()
+
+    /**
+     * The soft keyboard sends `tab` events when `submit` is clicked. This adds an extra step
+     * to many user flows (for example, on google.com, people expect a search to kick off after
+     * clicking submit, but it doesn't). This script forces a `submit` call after tab events are
+     * received.
+     */
+    val ADD_SUBMIT_LISTENER_TO_ALL_INPUTS = """
+        |if (typeof MOZ_FFTV_inputListener === 'undefined') {
+        |   MOZ_FFTV_inputListener = (event) => {
```

Comment:
nit: we're using `_firefoxTV_` as the prefix elsewhere in this file: can we use that instead of MOZ_FFTV?